File: app/src/main/java/xyz/hasnat/csdictionary/Model/Word.java
package xyz.hasnat.csdictionary.Model;

import java.util.ArrayList;

public class Word {
    private String word;
    private String definition;

    public Word(String word, String definition) {
        this.word = word;
        this.definition = definition;
    }

    public Word(String word, ArrayList<String> definition) {
        this.word = word;
        StringBuilder stringBuilder = new StringBuilder();
        for (String s : definition){
            stringBuilder.append(s);
        }
        this.definition = stringBuilder.toString();
    }

    public String getWord() {
        return word;
    }

    public String getDefinition() {
        return definition;
    }


    @Override
    public String toString() {
        return "Word{" +
                "word='" + word + '\'' +
                ", definition='" + definition + '\'' +
                '}';
    }
}

File: app/src/main/java/xyz/hasnat/csdictionary/model/Word.kt
package xyz.hasnat.csdictionary.model

import java.util.ArrayList

class Word {
    var word: String? = null
        private set
    var definition: String? = null
        private set

    constructor(word: String, definition: String) {
        this.word = word
        this.definition = definition
    }

    constructor(word: String, definition: ArrayList<String>) {
        this.word = word
        val stringBuilder = StringBuilder()
        for (s in definition) {
            stringBuilder.append(s)
        }
        this.definition = stringBuilder.toString()
    }


    override fun toString(): String {
        return "Word{" +
                "word='" + word + '\''.toString() +
                ", definition='" + definition + '\''.toString() +
                '}'.toString()
    }
}
